rest_api/src/main/resources/static/assets/js/jobs.js

```diff
@@ -0,0 +1,152 @@
+//var base_api_url = "http://192.168.160.63:8080";
+var base_api_url = "http://localhost:8080";
```

Comment:
This doesn't work on production. See this -> 8a6243ff939d42105798e550b21e377bd508e232